### ISSUE TITLE
https://callstack.github.io/react-native-paper is not working

### ISSUE THREAD

**Issue by AritraStark** [callstack/react-native-paper#4264]
### Current behaviour
Site is broken

### Expected behaviour
The website to work

### How to reproduce?
Visit: https://callstack.github.io/react-native-paper

### What have you tried so far?
Refreshing, opening it in multiple devices, refreshing again.

**Comment by lukewalczak** [comment]
Hey @AritraStark, what exactly does not work? I can open the website without any issues.

**Comment by AritraStark** [comment]
Might be regional server issue, the site isn't working for anyone nearby as well.

**Comment by lukewalczak** [comment]
I confirmed with couple folks the site works as expected, looks like there is an internal issue on your side, hence closing the issue.

**Comment by Kimbohlovette** [comment]
@AritraStark, I agree with you. It's a regional server issue. It does not open in my country with mobile data, but when I use vpn it opens
